tdrs-backend/manifest.yml

```diff
@@ -1,8 +1,20 @@
 version: 1
 applications:
 - name: tdp-backend
-  memory: 256M
+  memory: 512M
   instances: 1
   disk_quota: 2G
   docker:
     image: ((docker-backend))
+  env:
+    AV_SCAN_URL: http://((cf-space))-clamav-rest.apps.internal:9000/scan
+- name: clamav-rest
+  instances: 1
+  memory: 2G
+  disk_quota: 2G
+  docker:
+    image: rafttech/clamav-rest:0.103.2
+  env:
+    MAX_FILE_SIZE: 200M
```

Comment:
this is interesting. will this throw out any files larger than 200MB? this is quite a big larger than what we think we'll get so we may want to consider bringing this down a bit based on what this intended to do. @jtwillis92

Reply:
Yes, Clam AV will refuse to scan files larger than that limit. By default ClamAV will not scan files larger than 20Mb so this is actually raising the default here. We could definitely bring this down to match the expected max file size from actual data.